jenkins/shared-library/vars/githubHelper.groovy

```diff
@@ -103,11 +103,19 @@ void configureGitHub() {
 	runScript('git config user.email "jenkins@symbol.dev"')
 }
 
-void executeGitAuthenticatedCommand(Closure command) {
+void withGitHubToken(Closure closure) {
 	withCredentials([usernamePassword(credentialsId: helper.resolveGitHubCredentialsId(),
 			usernameVariable: 'GITHUB_USER',
 			passwordVariable: 'GITHUB_TOKEN')]) {
-		final String replaceUrl = "https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/.insteadOf https://github.com/"
+		closure()
+	}
+}
+
+void executeGitAuthenticatedCommand(Closure command) {
+	withGitHubToken {
+		final String ownerName = helper.resolveOrganizationName()
+		final String replaceUrl = 'https://$GITHUB_USER:$GITHUB_TOKEN@github.com/' +
```

Comment:
did you intentionally change from ${GITHUB_USER} to $GITHUB_USER ?

Reply:
Yeah, it's to ensure that Jenkins does not try to inject the values before they are executed by Bash.